src/models/event.ts

```diff
@@ -77,7 +77,6 @@ export interface IUnsigned {
     "invite_room_state"?: StrippedState[];
     "m.relations"?: Record<RelationType | string, any>; // No common pattern for aggregated relations
     [UNSIGNED_THREAD_ID_FIELD.name]?: string;
-    [UNSIGNED_MEMBERSHIP_FIELD.name]?: Membership | string;
```

Comment:
If I try to add both the stable and unstable property names here, the TypeScript checker complains about "A computed property name in an interface must refer to an expression whose type is a literal type or a 'unique symbol' type."  Removing the properties, and using the `.findIn` method seems to make it happy.